drgn/commands/_builtin/crash/ps.py

```diff
@@ -0,0 +1,294 @@
+                "-l",
+                dest="last_run_timestamp",
+                action="store_true",
+                default=False,
+                help="show last run information (raw timestamp)",
+            ),
+        ),
+        argument(
+            "-G",
+            dest="group_leader",
+            action="store_true",
+            default=False,
+            help="display only the thread group leader in a thread group",
+        ),
+        argument(
+            "arg",
+            nargs="?",
+            type=check_arg_type,
+            metavar="pid | task | command",
+            help="pid is a process PID. task is hexadecimal task_struct pointer. command is a command name.",
+        ),
+        drgn_argument,
+    ),
+)
+def _crash_cmd_ps(
+    prog: Program, name: str, args: argparse.Namespace, **kwargs: Any
+) -> None:
+
+    if args.drgn:
+        builder = CrashDrgnCodeBuilder(prog)
+        builder.add_from_import("drgn.helpers.linux.pid", "for_each_task")
+        builder.add_from_import("drgn.helpers.common.format", "escape_ascii_string")
+        builder.add_from_import(
+            "drgn.helpers.linux.mm",
+            "task_vsize",
+            "get_task_rss_info",
+            "totalram_pages",
+        )
+        builder.add_from_import(
+            "drgn.helpers.linux.sched", "task_state_to_char", "task_cpu"
+        )
+        if args.last_run or args.last_run_timestamp:
+            builder.add_from_import(
+                "drgn.helpers.linux.sched", "task_since_last_arrival_ns"
+            )
+            builder.add_import("datetime")
+
+        builder.append("total_mem = int(totalram_pages(prog))\n")
+        builder.append("page_size = prog['PAGE_SIZE'].value_()\n")
+
+        if args.user or args.kernel:
+            builder.add_from_import("drgn.helpers.linux.kthread", "is_kthread")
+            if args.user:
+                builder.append(
+                    "tasks = filter(lambda t: not task_is_kthread(t), for_each_task(prog))\n"
+                )
+            elif args.kernel:
+                builder.append("tasks = filter(task_is_kthread, for_each_task(prog))\n")
+        else:
+            builder.append("tasks = for_each_task(prog)\n")
+
+        if args.group_leader:
+            builder.add_from_import(
+                "drgn.helpers.linux.sched",
+                "thread_group_leader",
+            )
+            builder.append("tasks = filter(thread_group_leader, tasks)\n")
+
+        builder.append("for task in tasks:\n")
+        builder.append("    cpu = task_cpu(task)\n")
+        builder.append("    state = task_state_to_char(task)\n")
+        builder.append("    task_rss = get_task_rss_info(task)\n")
+        builder.append("    total_rss = task_rss.total\n")
+        builder.append("    pct_mem = total_rss * 100 / total_mem\n")
+        builder.append("    vmem = task_vsize(task)\n")
+        builder.append("    command = escape_ascii_string(task.comm.string_())\n")
```

Comment:
No need to convert this to a string here, just

```suggestion
        builder.append("    command = task.comm\n")
```
(And you can remove the `add_from_import` for `escape_ascii_string`).